sdk/src/types/block/capabilities.rs

```diff
@@ -58,6 +58,15 @@ impl<Flag> Capabilities<Flag> {
     }
 }
 
+impl<Flag> core::fmt::Debug for Capabilities<Flag> {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("Capabilities")
+            .field("bytes", &self.bytes)
+            .field("flag", &std::any::type_name::<Flag>().rsplit_once("::").unwrap().1)
```

Comment:
sad `no_std` noises

Reply:
just add a `Debug` bound for the flag and call that

Reply:
^^^

Reply:
how can I call that if there's no instance of it? it's phantom data, or do you mean something else?